MainRepository/Program.cs

```diff
@@ -1,7 +1,28 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿//Program to find the frequency of alphabets in a dictionary.
+namespace Test2 {
+   class LetterFrequency {
+      public static void Main (string[] args) {
+         string[] words = File.ReadAllLines ("c:/etc/words.txt");
+         List<char> Letters = new List<char> ();

```

Comment:
```suggestion
         List<char> letters = new List<char> ();

```
Variable names should be in lower case